dom.css

```diff
@@ -0,0 +1,151 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  width: 100%;
+  height: 100%;
+  margin-left: auto;
+  margin-right: auto;
+}
+/* style header section */
+.header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  font-size: 17px;
+  border: 1px solid black;
+  text-align: end;
+  width: 100%;
+  height: 50px;
+  background: #f1f1f1;
+  padding: 10px;
+}
+.header a {
+  font-size: 20px;
+  font-weight: 600;
+}
+
+/* Style the submit button */
+form.example button {
+  float: left;
+  width: 20%;
+  padding: 10px;
+  background: #2196f3;
+  color: white;
+  font-size: 17px;
+  border: 1px solid black;
+  border-left: none; /* Prevent double borders */
+  cursor: pointer;
+}
+
+/* About section */
+.about {
+  padding: 10px;
+  font-size: 16px;
+  font-weight: 400;
+  border: 1px solid black;
+  width: 80%;
+  height: 150px;
+  background: #f1f1f1;
+  margin-top: 60px;
+}
+/* Experience section */
+#experiences {
+  border: 1px solid black;
+  background: #f1f1f1;
+  padding: 10px;
+  width: 80%;
+  height: 200vh;
+  max-height: 300px;
+  overflow-y: auto;
+  margin-top: 10px;
+}
+/* style the experience button */
+#addExperience {
+  float: right;
+  width: 10%;
+  padding: 10px;
+  background: #2196f3;
+  color: white;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+  margin-top: -60px;
+}
+#description {
+  font-size: 16px;
+  font-weight: 400;
+  border: 1px solid black;
+  width: 100%;
+  background: #f1f1f1;
+  margin-top: 20px;
+}
+/* style addExperienceForm button */
+#addExperienceForm button {
+  float: right;
+  width: 10%;
+  padding: 10px;
```

Comment:
Also instead of float it is better approach to use flexbox or grid. You should learn a little about that as well. For now float works.